server/app/views/components/ToastFragment.html

```diff
@@ -14,6 +14,22 @@
   ></div>
 </div>
 
+<div
+  th:fragment="alert(id, message, canDismiss, canIgnore, toastDuration, condOnStorageKey)"
+>
+  <div
```

Comment:
question (non-blocking):  Can we include `role="alert"` or `aria-live` so that the toasts will be a11y compliant?

Reply:
that's actually handled by toast.ts. It grabs the content from these toasts and renders it to the user with role and aria-live specified.

Reply:
Unfortunately, we will need some amount of scripting for toasts to handle features like condOnStorageKey and ignore lists and dismissing after a certain period of time. I don't particularly like relying on toast.ts mainly because it applies old non-USWDS styles instead of making things look like a USWDS alert, but I have an issue filed for that work. https://github.com/civiform/civiform/issues/7747